server/portal/libs/agave/utils.py

```diff
@@ -222,6 +222,15 @@ def service_account():
         access_token=settings.TAPIS_ADMIN_JWT)
 
 
+def user_account(access_token, refresh_token):
```

Comment:
Typically we access Tapis clients for users by retrieving their user model from Django and doing `<user>.tapis_oauth.client`. Instantiating clients with explicit access/refresh tokens would be a new pattern, so it requires a strong justification IMO.